### DISCUSSION TITLE
How change md style

### DISCUSSION THREAD

**Discussion by faye1225** [steven-tey/novel#288] [Q&A]
I'd like to make some adjustments for the editor style. `h3, p, blockquote` tags, etc. The tech stack I'm using is `nextjs+ts+tailwindcss`

My code looks like this, but it doesn't work. How should I change it properly?
```
import { Editor } from 'novel'
import { Node } from '@tiptap/core'

const CustomBlockquote = Node.create({
  name: 'blockquote',
  addOptions() {
    return {
      HTMLAttributes: {
        class: 'border-l-1 '
      }
    }
  }
})

export const NovelEditor = (props: NovelEditorProps) => {
  return (
    <Editor
      defaultValue={props.defaultValue}
      disableLocalStorage
      extensions={[CustomBlockquote]}
    />
  )
}
```

**Comment by mansidak** [comment]
Hey @faye1225! Did you ever figure this out? Stuck on somethign similar.

**Comment by andrewdoro** [comment]
You can now customize each extension
https://novel.sh/docs/guides/tailwind/extensions
